modules/microshift-multus-intro.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-intro_{context}"]
+= Additional networks in {microshift-short}
+
+During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the Multus CNI plugin, you can define _additional networks_ based on available CNI plugins. You can then attach one or more of these networks to your pods. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.
```

Comment:
> Using the Multus CNI plugin, you can define _additional networks_ based on available CNI plugins. You can then attach one or more of these networks to your pods

I wouldn't say that using Multus CNI you can define additional networks.
It's true to some extent about `bridge` plugin (because it can create the bridge if it doesn't exist) but other plugins connect to existing networks, like macvlan or ipvlan can be used to connect Pod directly to host's network.
To me, it's more about adding additional interfaces to Pod from (mostly) existing networks

Reply:
Recast the language.